pkg/credentials/s3/s3_secret.go

```diff
@@ -49,6 +50,7 @@ var (
 	InferenceServiceS3SecretHttpsAnnotation      = constants.KFServingAPIGroupName + "/" + "s3-usehttps"
 	InferenceServiceS3UseVirtualBucketAnnotation = constants.KFServingAPIGroupName + "/" + "s3-usevirtualbucket"
 	InferenceServiceS3UseAnonymousCredential     = constants.KFServingAPIGroupName + "/" + "s3-useanoncredential"
+	InferenceServiceS3CABundlePathAnnotation     = constants.KFServingAPIGroupName + "/" + "s3-cabundlepath"
```

Comment:
`s3-cabundlepath` -> `s3-cabundle` just to be consistent with the env variable name?

Reply:
I was unsure about naming, will change it :)